deepchecks/tabular/context.py

```diff
@@ -102,7 +219,11 @@ def __init__(self,
         if features_importance is not None:
             if not isinstance(features_importance, pd.Series):
                 raise DeepchecksValueError('features_importance must be a pandas Series')
-
+        # we do it after validation because the model we are creating isn't valid (we are always valid just not here)
```

Comment:
>  we are creating isn't valid 

what's not valid about him?
>  (we are always valid just not here)

not sure what you mean

Reply:
dummy model isn't of type base estimator so it will fail with our validation
"(we are always valid just not here)" this one I tried to be funny

Reply:
ok it isn't needed